driver/camera.c

```diff
@@ -447,9 +447,27 @@ static void i2s_init()
     I2S0.timing.rx_dsync_sw = 1;
 
     // Allocate I2S interrupt, keep it disabled
-    esp_intr_alloc(ETS_I2S0_INTR_SOURCE,
-                   ESP_INTR_FLAG_INTRDISABLED | ESP_INTR_FLAG_LEVEL1 | ESP_INTR_FLAG_IRAM,
```

Comment:
You can pass `ESP_INTR_FLAG_LOWMED` instead of `ESP_INTR_FLAG_LEVEL1` — the effect will be the same (pick the lowest level interrupt which is available)

Reply:
Thanks!